tests/system/model_monitoring/test_model_monitoring.py

```diff
@@ -1050,7 +1051,9 @@ def _test_feature_names(self) -> None:
         feature_names = [feat.name for feat in features]
         assert feature_names == [
             mlrun.feature_store.api.norm_column_name(feat)
-            for feat in self.columns + [self.y_name]
+            for feat in itertools.chain(
+                self.columns, [self.y_name], mm_constants.FeatureSetFeatures.list()
+            )
```

Comment:
`itertools` is unnecessary:
```suggestion
            for feat in self.columns + [self.y_name] + mm_constants.FeatureSetFeatures.list()
```